src/Hl7.Fhir.Base/Serialization/engine/IFhirSerializationEngine.cs

```diff
@@ -73,7 +73,7 @@ public static class SerializationEngineExtensions
         /// </summary>
         /// <exception cref="InvalidOperationException">Thrown if the underlying engine is a legacy engine</exception>
         /// <exception cref="DeserializationFailedException">Thrown if a FHIR error was encountered in the data</exception>
-        public static Resource? SerializeReaderToXml(this IFhirSerializationEngine engine, XmlReader reader)
+        public static Base? SerializeReaderToXml(this IFhirSerializationEngine engine, XmlReader reader)
```

Comment:
This is a breaking change, can we keep this change internal? Otherwise, we'll need to add a comment to the PR that this is a breaking change.

Reply:
Ok, so a breaking change, make sure to put it in the PR.